### PR TITLE
Allow disabling securityContext

### PR DESCRIPTION
**Pull request by jradtilbrook**
**What this PR does / why we need it**: allow disabling securityContext on the pod

**Which issue this PR fixes**: relates to #39

**Special notes for your reviewer**:

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by rimusz** [comment]
Can you merged #49 to this PR as well, thanks.
Also can you change `securityContext` to be applied as you did for the pod annotations, that will allow to set e.g. user, group and etc. Keep privileged as an example, but comment it out.

**Review by rimusz — CHANGES_REQUESTED** [review]
Can you merged #49 to this PR as well, thanks.
Also can you change securityContext to be applied as you did for the pod annotations, that will allow to set e.g. user, group and etc. Keep privileged as an example, but comment it out.

File: stable/agent/values.yaml [hunk 1]
```diff
@@ -22,8 +22,7 @@ enableHostDocker: true
 
 # Enable security context configuration
 securityContext:
```
Comment (rimusz) [inline stable/agent/values.yaml]: if nothing set then there should be `securityContext: {}`

Comment (jradtilbrook) [reply stable/agent/values.yaml]: I wondered that but then saw the `extraEnv` doesn't have a default like that

Comment (rimusz) [reply stable/agent/values.yaml]: as `extraEnv` is not part of the template spec, but `securityContext` is.

Comment (jradtilbrook) [reply stable/agent/values.yaml]: Oh okay thanks for clearing that up

Comment (rimusz) [reply stable/agent/values.yaml]: sorry my bad we have `{{- if .Values.securityContext.enabled }}` so it has to be `securityContext:`.
sorry for confusion.

Comment (jradtilbrook) [reply stable/agent/values.yaml]: But this PR changes that to use the `with` syntax so the `securityContext.enabled` property is no longer used. I think I've fixed it with the last push

**Pull request merged**